lib/matplotlib/backend_bases.py

```diff
@@ -1409,9 +1409,12 @@ class MouseEvent(LocationEvent):
 
     step : scalar
         The number of scroll steps (positive for 'up', negative for 'down').
+        This applies only to 'scroll_event' and defaults to 0 otherwise.
 
     dblclick : bool
-        Whether the event is a double-click.
+        Whether the event is a double-click. This applies only to
+        'button_press_event' and is False otherwise. In particular, it's
+        not used in 'button_release_event'.
```

Comment:
"not used" -> "never set"?

Reply:
Mhh, these don't really match my intention. "never set" and "always False" both sound like the value is still meaningful but has a fixed value. I really would like to emphsize that for button_release_event `dblclick` is not part of the logic/API.

Reply:
It's not clear to me whether "not used" means "the attribute does not exist" or "the attribute exists but can take any value, which is meaningless" (well I guess that's more a problem in C (uninitialized variable)) or "is always False".
Anyways, it's not a big deal, feel free to self-merge if you're not going to change this.

Reply:
Going to merge as is since I don't have an idea how to express this more clearly without getting very verbose. Anyway, it's an improvement.